cayenne-server/src/test/java/org/apache/cayenne/access/JointPrefetchIT.java

```diff
@@ -401,4 +402,52 @@ public void testJointPrefetchMultiStep() throws Exception {
             assertEquals(PersistenceState.COMMITTED, g2.getPersistenceState());
         });
     }
+
+    @Test
+    public void testJointPrefetchSQLSelectToMany() throws Exception {
+        createJointPrefetchDataSet();
+        SQLSelect sqlSelect = SQLSelect.query(Artist.class, "SELECT "
```

Comment:
This and later code can be nicer if `sqlSelect` will be defined as `SQLSelect<Artist>` and you can use chained method calls. We should keep even test code clear, as it may serve as example.